docker-compose.yml

```diff
@@ -216,7 +216,7 @@ services:
       - "9114:9114"
 
   keycloak:
-    image: quay.io/keycloak/keycloak:18.0.2
+    image: quay.io/keycloak/keycloak:21.0.2
```

Comment:
Just wanted to double check that this worked?   I think this is the jump to Quarkus based JVM from the old style right?   It loaded these files and just ran?

Reply:
IIRC we already did the jump to the Quarkus based JVM with one of the last version bumps as Keycloak 17 was the first version to use Quarkus by default (see https://www.keycloak.org/migration/migrating-to-quarkus) and we're already on 18.
What I did to test the component updates is run `./quickstart.sh -vector -lab` and verify that I can generate an account through Keycloak and use it to sign into Quepid which worked.
As Quepid is the only component using Keycloak in Chorus ES (if I'm not missing anything essential right now) we should be good.